index.html

```diff
@@ -0,0 +1,254 @@
+      margin: auto;
+    }
+
+    ul {
+      list-style: none;
+    }
+
+    img {
+      width: 100%;
+      height: auto;
+      display: block;
+    }
+
+    #header {
+      background: #00002a;
+      height: auto;
+      width: 100%;
+      padding: 20px 0 80px 0;
+    }
+
+    #nav-bar {
+      display: flex;
+      align-items: center;
+      position: fixed;
+      float: right;
+      margin-left: 65%;
+      top: 0;
+  
+    }
+
+    #header-img {
+      font-size: 1.5em;
+      font-weight: 700;
+      color: #fff;
+      width: 50px;
+      height:50px;
+    
+    }
+
+    nav ul {
+      display: flex;
+      align-items: center;
+      width: fit-content;
+    }
+
+    .nav-link {
+      display: block;
+      padding: 0px 10px;
+      color: #fff;
+      text-decoration: none;
+      font-size: 1.1em;
+      cursor: pointer;
+    }
+
+    .head-flex {
+      margin: 90px auto 20px auto;
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+    }
+
+    .flex-text {
+      width: calc(50% - 40px);
+    }
+
+    .flex-img {
+      width: 50%;
+      height: 200px;
+      background: #9f9f9f;
+      overflow: hidden;
+    }
+
+    .flex-text h2 {
+      font-size: 2.2em;
+      text-align: left;
+      color: #fff;
+      font-weight: 800;
+    }
+
+    .flex-text p {
+      text-align: left;
+      font-size: 1em;
+      color: #fff;
+      margin: 10px 0;
+    }
+
+    .flex-text button {
+      outline: none;
+      border: none;
+      border-radius: 5px;
+      background: skyblue;
+      color: #fff;
+      padding: 5px 15px;
+      cursor: pointer;
+    }
+
+    #r {
+      background: #00002a;
+      height: auto;
+      width: 100%;
+      padding: 20px 0 80px 0;
+    }
+
+    .cards {
+      width: 90vw;
+      margin: 3rem auto;
+      display: grid;
+      gap: 1rem;
+      direction: rtl;
+    }
+
+    @media screen and (min-width: 900px) {
+      .cards {
+        grid-template-columns: repeat(3, 1fr);
+      }
+    }
+
+    @media screen and (min-width: 640px) and (max-width: 900pxx) {
+      .cards {
+        grid-template-columns: repeat(2, 1fr);
+      }
+    }
+
+    .card {
+      border-radius: 8px;
+      background-color: white;
+      padding: 0.7rem;
+      direction: rtl;
+      box-shadow: rgba(50, 50, 93, 0.25) 0px 2px 5px -1px, rgba(0, 0, 0, 0.3) 0px 1px 3px -1px;
+    }
+  </style>
+</head>
+
+<body>
+  <h1 id="header">
+    <img id="header-img" src="https://i.pinimg.com/564x/a1/87/87/a187875e7d7a9cf891f3868687e9151a.jpg"
+      alt="Company Logo" title="" />
+    
+
+    <nav class="container" id="nav-bar">
+      <ul>
+        <li><a href="#info" class="nav-link">Info</a></li>
+        <li><a href="#about" class="nav-link">About</a></li>
+        <li><a href="#videos" class="nav-link">Videos</a></li>
+      </ul>
+    </nav>
+
+
+    <section class="head-flex container" id="about">
+      <div class="flex-text">
+        <h2>This website is awesome</h2>
```

Comment:
The inline comment something like this in the lines of code that needs to changed